src/tests/JIT/Regression/JitBlue/Runtime_95315/Runtime_95315.cs

```diff
@@ -0,0 +1,56 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using Xunit;
+
+public class Runtime_95315
+{
+    [Fact]
+    public static void TestEntryPoint()
+    {
+        for (int i = 0; i < 4; i++)
+        {
+            for (int j = 0; j < 200; j++)
+            {
+                Assert.Throws<IndexOutOfRangeException>(() => Bar(new int[10], new C()));
+            }
+
+            Thread.Sleep(100);
+        }
+
+        Assert.Throws<IndexOutOfRangeException>(() => Bar(new int[1], new C()));
+    }
+
+    [MethodImpl(MethodImplOptions.NoInlining)]
+    private static void Bar(int[] arr, I iface)
+    {
+        if (arr == null)
+        {
+            return;
+        }
+
+        iface.Foo();
+
+        int i = 10000000;
```

Comment:
why 10M? presumably it will fail after 10 iterations? (if it doesn't fail, we don't want the test writing 10MB of data)

Reply:
I picked a large constant that has high probability to hit AV if we wrongly elide the check. But you're right, we shouldn't be doing `Console.WriteLine` with it, will fix that

Reply:
Note that `arr.Length` is only 1

Reply:
On second thought, if we hit valid data at + 10000000 then we only print one time anyway, since `i < arr.Length` will immediately be false.

Reply:
Going to leave it like this since it'll at most print one time.